index.html

```diff
@@ -45,12 +45,18 @@
       <h1 class="text-center mb-5">Hello fellow!</h1>
       
       <h2 class="mb-5">This is a page where you are able to learn current information about mobilization in Ukraine!</h2>
+      <div id="marquee">
```

Comment:
are you really want to have this runnig string? I my point of view its oanly chane fokus from form to string.. may be is better in feature to change this to some banner on image wichout any effect.. but it's on u :)